avm/ptn/aca-lza/hosting-environment/modules/spoke/deploy.spoke.bicep

```diff
@@ -0,0 +1,606 @@
+          destinationAddressPrefix: '*'
+          destinationPortRange: '65200-65535'
+          access: 'Allow'
+          priority: 100
+          direction: 'Inbound'
+        }
+      }
+      {
+        name: 'Allow_TLS'
+        properties: {
+          description: 'allow https incoming connections'
+          protocol: '*'
+          sourceAddressPrefix: '*'
+          sourcePortRange: '*'
+          destinationAddressPrefix: '*'
+          destinationPortRange: '443'
+          access: 'Allow'
+          priority: 110
+          direction: 'Inbound'
+        }
+      }
+      {
+        name: 'Allow_HTTP'
+        properties: {
+          description: 'allow http incoming connections'
+          protocol: '*'
+          sourceAddressPrefix: '*'
+          sourcePortRange: '*'
+          destinationAddressPrefix: '*'
+          destinationPortRange: '80'
+          access: 'Allow'
+          priority: 120
+          direction: 'Inbound'
+        }
+      }
+      {
+        name: 'Allow_AzureLoadBalancer'
+        properties: {
+          description: 'allow AzureLoadBalancer incoming connections'
+          protocol: '*'
+          sourceAddressPrefix: 'AzureLoadBalancer'
+          sourcePortRange: '*'
+          destinationAddressPrefix: '*'
+          destinationPortRange: '80'
+          access: 'Allow'
+          priority: 130
+          direction: 'Inbound'
+        }
+      }
+      {
+        name: 'allow-all-outbound'
+        properties: {
+          protocol: '*'
+          sourcePortRange: '*'
+          destinationPortRange: '*'
+          access: 'Allow'
+          priority: 210
+          direction: 'Outbound'
+          sourceAddressPrefix: 'VirtualNetwork'
+          destinationAddressPrefix: '*'
+        }
+      }
+    ]
+    diagnosticSettings: [
+      {
+        name: 'logAnalyticsSettings'
+        workspaceResourceId: logAnalyticsWorkspace.outputs.resourceId
+      }
+    ]
+  }
+}
+
+@description('NSG Rules for the private enpoint subnet.')
+module nsgPep 'br/public:avm/res/network/network-security-group:0.2.0' = {
+  name: take('nsgPep-${deployment().name}', 64)
+  scope: resourceGroup(rgSpokeName)
+  params: {
+    name: naming.outputs.resourcesNames.pepNsg
+    location: location
+    tags: tags
+    securityRules: [
+      {
+        name: 'deny-hop-outbound'
+        properties: {
+          protocol: '*'
+          sourcePortRange: '*'
+          destinationPortRanges: [
+            '3389'
+            '22'
+          ]
+          access: 'Deny'
+          priority: 200
+          direction: 'Outbound'
+          sourceAddressPrefix: 'VirtualNetwork'
+          destinationAddressPrefix: '*'
+        }
+      }
+    ]
+    diagnosticSettings: [
+      {
+        name: 'logAnalyticsSettings'
+        workspaceResourceId: logAnalyticsWorkspace.outputs.resourceId
+      }
+    ]
+  }
+}
+
+//TODO: This needs to be replaced once the peering module is available in the avm modules
```

Comment:
Either that, or you deploy the peering directly via the VNET module. It allows you to deploy both peering-directions if desired

Reply:
Moved to vnet.